src/components/pages/devices/devices.js

```diff
@@ -54,6 +54,20 @@ class DevicesPage extends Component {
   /** Listen for changes in the dynamic context filters and update accordingly */
   onContextMenuChange = contextBtns => this.setState({ contextBtns });
 
+  filterDevies(devices) {
+    if (!devices || !devices.Items) {
+      return devices;
+    }
+    return {
+      Items: devices.Items.filter((device) => {
+        if (!this.props.params || !this.props.params.status) {
+          return true;
+        }
+        return device.Connected === (this.props.params.status === 'Connected');
```

Comment:
Is 'Connected ' going to be case-sensitive? Can you just compare with case-insensitive?

Reply:
Yes. 
![image](https://user-images.githubusercontent.com/16091688/35119763-cc91ab84-fc4a-11e7-85dc-7040164a4766.png)

Reply:
My point was if you can do something like this this.props.params.status.toLower() === 'connected' Then we don't have to worry about future scenario where case changes